docker-compose-config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source env/bin/activate 2> /dev/null || source venv/bin/activate 2> /dev/null # look for a virtual env
```

Comment:
This doesn't work for example with pyenv.

Reply:
Given that users will be running this command in a vanilla checkout, I think we should either drop this or rethink it.